### DISCUSSION TITLE
Show more than 5 entries on relationship fields

### DISCUSSION THREAD

**Discussion by TiiFuchs** [laravel/nova-issues#3905] [Q&A] [answered]
When using a Relationship Field (with more than one relationship models) on a Resource, it is displayed as table below with 5 entries per page.

It would be cool to be able to define how much entries per page should be visible there. I didn't find any way to do this (the easy way) right now.

**Comment by pcorrick** [comment, marked as answer]
I believe you can set this on your Resource class, for example:
`public static $perPageViaRelationship = 10;`

**Reply by TiiFuchs** [reply]
Oh thanks. I'll try this later. :)
